src/Models/StoredEventData.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Spatie\EventProjector\Models;
+
+use Exception;
+use Illuminate\Support\Arr;
+use Illuminate\Contracts\Support\Arrayable;
+use Spatie\EventProjector\Facades\Projectionist;
+use Spatie\EventProjector\Exceptions\InvalidStoredEvent;
+use Spatie\EventProjector\EventSerializers\EventSerializer;
+
+class StoredEventData implements Arrayable
```

Comment:
Since `StoredEventData` is what package users will be interacting with, I'd keep it as `StoredEvent`, and rename the Eloquent model since it's an internal implementation detail.

Reply:
I was thinking this too, but renaming the model would mean a breaking change, which currently this PR isn't

Reply:
Renamed it